From 02412cd52c10722562c79c7ac1fcbbc95d3424d1 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 02 Apr 2020 00:01:13 +0300
Subject: [PATCH] [SW-0] preboot: fix u-boot spi offset printed

Current Preboot code prints the U-Boot SPI flash offset base (stable or
candidate).
The real U-Boot offset on SPI flash includes the base...
So print the real U-Boot offset for more precision and more information.

Change-Id: If646e5c77c6df80bb799f7dc362aee01d07e2c97
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/app/preboot.c b/app/preboot.c
index ff4410c..1da59bc 100644
--- a/app/preboot.c
+++ b/app/preboot.c
@@ -87,7 +87,7 @@
 
 	preboot_puts_raw("Preparing to run U-Boot ");
 	preboot_puts_raw(uboot_image);
-	preboot_puts_hex_raw(" (offset: ", img_offset, false);
+	preboot_puts_hex_raw(" (offset: ", uboot_offset, false);
 	preboot_puts(")\n");
 
 	spi_copy((void *)UBOOT_TEXT_BASE, uboot_offset, UBOOT_SIZE, name);
